webhub/models.py

```diff
@@ -50,241 +48,54 @@ class Pcuser(models.Model):
     def __unicode__(self):
         return self.user.username
 
+# Post table stores details about posts
 
-#Post table stores details about posts
 
 class Post(models.Model):
-    #The owner of the post
+    # The owner of the post
     owner = models.ForeignKey(Pcuser, null=False, related_name='owner')
-    #title
-    title_post = models.CharField(max_length=300)
-    #description
-    description_post = models.CharField(max_length=2000)
-    #link to important documents
+    title_post = models.CharField(max_length=100,
+                                  validators=[
```

Comment:
great job adding input sanitization .. should we not sanitize the other inputs as well? user location  phone e.t.c..since there might be some special characters that can break the application ?